address-registry.json

```diff
@@ -0,0 +1,505 @@
+{
+    "chains": [
+        {
+            "name": "ETH",
+            "contracts": {
+                "actors": [
+                    {
+                        "name": "governor",
```

Comment:
Noticed some contracts don't have labels, should we include them so the `README` is more readable?

Reply:
You mean the protocol-registry md file?

Reply:
Yeah

Reply:
I think that maybe the proxy would make sense, although I don't think it's super important. Honestly, it's a bit cumbersome to handle the labels on generation, but if you both think it's worth it, I can do it.